code/modules/lighting/lighting_atom.dm

```diff
@@ -99,8 +99,11 @@
 	cut_overlay(exposure_overlay)
 
 	if(glow_icon && glow_icon_state)
-		glow_overlay = image(icon = glow_icon, icon_state = glow_icon_state, dir = dir, layer = 1)
-		glow_overlay.plane = LIGHTING_LAMPS_PLANE
+		glow_overlay = image(icon = glow_icon, icon_state = glow_icon_state, dir = dir, layer = -2)
```

Comment:
Does changing the layer here impact how glare works for other light types?

Reply:
Nope